examples/jsm/renderers/webgpu/WebGPURenderer.js

```diff
@@ -157,7 +157,8 @@ class WebGPURenderer {
 
 		const device = await adapter.requestDevice( deviceDescriptor );
 
-		const glslang = await import( 'https://cdn.jsdelivr.net/npm/@webgpu/glslang@0.0.15/dist/web-devel/glslang.js' );
+		// https://cdn.jsdelivr.net/npm/@webgpu/glslang@0.0.15/dist/web-devel/glslang.js
+		const glslang = await import( '../../libs/glslang.js' );
```

Comment:
I think this can now become a static import?

```js
// https://cdn.jsdelivr.net/npm/@webgpu/glslang@0.0.15/dist/web-devel/glslang.js
import glslang from '../../libs/glslang.js' 
```